plugins/exporter.koplugin/template/md.lua

```diff
@@ -39,6 +39,9 @@ local function prepareBookContent(book, formatting_options, highlight_formatting
     local content = ""
     local current_chapter = nil
     content = content .. "# " .. book.title .. "\n"
+    if not book.author then
+        book.author = "(Author not found in book's metadata)"
```

Comment:
This probably doesn't need to be that verbose ? a `N/A` like we have in Book information could also be fine (and less eye catching) in exports ?
Also, this or `_("N/A")` should be `_()` wrapped so they can be translated.

Reply:
Also, I don't (want to) know where the `book` table/object comes from, and if it can have a life upper in our caller - and there, you are updating this object, which might not be very polite :) and could cause issues if upper code would use it ie. for comparing.
So, best to use a private local variable for that: `local author = book.author or _("N/A")`

Reply:
Changed to a less verbose version, but personally it might confuse me to see N/A without explanations.
I agree with consistency tho!
Should I push the N/A version with the parenthesis fix? What does that syntax mean in lua?

![image](https://user-images.githubusercontent.com/98263539/229299145-df6de3e6-5ffb-408f-a4b2-2fa2d706a99b.png)

Reply:
Thanks for catching the mutation to the table! I totally agree, that's not very polite 😅😂 Fixed it with your suggestion :)

Reply:
https://en.wikipedia.org/wiki/N/A
Looks like it's not translated in Italian. In French, it's translated to N/D (which to me is even more confusing :)
The fact that is it short also helps you not focusing on it. If you have `(Author not found in book's metadata)`, you'll be focusing and reading it to get to understand "ok, no author". Seeing the short N/A should get you to that more quickly?

> What does that syntax mean in lua?

All these do the same job, Lua allows for the short readable "natural english" versions:
```lua
local author
if book.author then
    author = book.author
else
    author = _("N/A")
end

local author = _("N/A")
if book.author then
    author = book.author
end

local author = book.author and book.author or _("N/A")

local author = book.author or _("N/A")
```

Reply:
Thanks for the explanations! :) 
Pushed with the fix! 🥳